Files/View Models/SelectedItemsPropertiesViewModel.cs

```diff
@@ -517,10 +525,18 @@ public void CheckForImage()
                     IsSelectedItemImage = true;
                     return;
                 }
+                else if (ItemExtension.Equals(".lnk", StringComparison.OrdinalIgnoreCase)
```

Comment:
Instead of checking the extension you can probably use `App.CurrentInstance.ContentPage.SelectedItem..IsShortcutItem`

Reply:
Also If you are using this to show "Open File Location" you may not want to do this for ".url" files

Reply:
I just added a new commit which makes use of `App.CurrentInstance.ContentPage.SelectedItem.IsShortcutItem`.
Url files are now also supported. This should now be the same behaviour as in the `PropertiesShortcut` Tab

Reply:
@tobias-mayer  i don't think it makes sense to show "open file location" for .url files, the only thing that option would do is open the shortcut, which by the way should be disabled in the properties window

Reply:
Ok yes I also think so. I'll change this. Should I also change it in the properties window or should I open another issue therfore?

Reply:
it would be better to add an issue and fix it in a new PR

Reply:
Now the option only shows up for ".lnk" files in the context menu